src/components/grid-aware/StatsBlock/StatsBlock.module.css

```
@@ -0,0 +1,92 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    background-color: var(--color-black);
+    grid-template-columns: 40px 1fr 40px;
+  }
+}
+
+/* Main grid */
+
+.gridParent {
+  background-color: var(--color-black);
+  color: var(--color-white);
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding-bottom: 150px;
+  padding-top: 100px;
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding-bottom: 38px;
+    padding-top: 50px;
+  }
+}
+
+.gridAreaBottom {
+  display: flex;
+  flex-direction: row;
+  grid-column-start: 2;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaBottom {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.title {
+  font: var(--font-title);
+  grid-column-end: 11;
+  grid-column-start: 2;
```

Nit: I think for most of this site, we've stuck to the start + span syntax for specifying the columns for components, since we've found it to be more intuitive. Note that what you wrote down here actually means "start at and include column 2, and go through but _do not include_ column 11", meaning that this is actually 9 columns wide.

Also, not nit: According to the designs, this should be 8 columns wide. This is more noticeable in the [Design System Figma](https://www.figma.com/file/HLVHC58KwkcVFXkKAuY0ej/ShelterTech.org-Design-System?node-id=808%3A14), where there's enough text in the title to span two rows.

```suggestion
  grid-column: 2 / span 8;
```